database.hh

```diff
@@ -1568,6 +1569,8 @@ public:
      */
     future<> clear_snapshot(sstring tag, std::vector<sstring> keyspace_names, const sstring& table_name);
 
+    future<std::unordered_map<sstring, std::unordered_map<std::pair<sstring, sstring>, db::snapshot_ctl::snapshot_details, utils::tuple_hash>>> get_snapshot_details();
```

Comment:
A comment describing the returned type would make it easier to understand the code.

Reply:
I was considering introducing an alias here with `using`, but wasn't quite sure whether we want to have this exported to other translation units (since it's a header file)

Reply:
I think a comment would be good enough

Reply:
So we don't get neither comments nor `using`? :(